### PR TITLE
minor usability: change position of "module order" tickbox in selective copy-paste dialog to the end (fixes #9671)

### LINKED ISSUE #9671: minor usability: position of "module order" tickbox in selective copy-paste dialog
**Is your feature request related to a problem? Please describe.**

I use a lot the selective copy-paste (Control-Shift-C) to copy just the last few modules I edited. Using the keyboard, I do:

1. Control-Shift-C to open the selective copy-paste dialog
2. Alt-n for "select none"
3. Space or Enter to re-select the module to copy
4. If needed, downarrow, space, as many times as there are modules to copy
5. Alt-O for "OK"

At step 3., with old versions of dt, I needed just a single press to space. Now that "module order" is also offered, I need one more down-arrow press. Not a big deal, but it used to be slightly more fluent when done with the keyboard only. However, having the option to tick/untick the module order is good.

**Describe the solution you'd like**

Moving the "module order" checkbox to the end of the list make it easier for keyboard users, and would probably also be convenient for mouse-users since it would make it easier to focus on the last edited modules, which is most likely the ones the user is interested in.

### PR DESCRIPTION
**Pull request by domosbg**
This fixes #9671. It was labeled with "good first contribution", so I though I try to fix it. Is it ok to make a pull request directly or should the issue first be assigned to the one who is working at the issue?

Can anyone take a look at my changes to see if it is ok.

Besides I have a question in relation to the selective copy-paste dialog. Is there a reason why the module order entry is not shown in the paste dialog when module order is not selected in the copy dialog?
All the other entries are also shown if they are not selected in the copy dialog and you have the possibility to activate modules for pasting which has not been selected in the copy dialog.

### CODE REVIEW

**Comment by johnny-bit** [comment]
Great! However please change wording or add `fixes #9671` since github parser doesn't understand proper English ;)

File: src/gui/hist_dialog.c [hunk 1]
```diff
@@ -40,15 +40,23 @@ typedef enum _style_items_columns_t
...
   gint num = 0;
-  if(gtk_tree_model_get_iter_first(model, &iter))
-    gtk_tree_model_get(model, &iter, DT_HIST_ITEMS_COL_ENABLED, &active, DT_HIST_ITEMS_COL_NUM, &num, -1);
-  return active && (num == -1);
+
+  gtk_tree_model_get_iter_first(model, &iter);
+  do
+  {
+      gtk_tree_model_get(model, &iter, DT_HIST_ITEMS_COL_ENABLED, &active, DT_HIST_ITEMS_COL_NUM, &num, -1);
+      if (active && (num == -1)) module_order_was_copied = TRUE;
```
Comment (TurboGit) [inline src/gui/hist_dialog.c:55]: `if (` to `if(`

**Review by TurboGit — CHANGES_REQUESTED** [review]
A very minor style change otherwise looks good. I haven't tested it yet/

File: src/gui/hist_dialog.c (hunk 1 above)
Comment (domosbg) [reply src/gui/hist_dialog.c]: Changed it as requested.

**Review by TurboGit — APPROVED** [review]
Works for me, thanks!

**Pull request merged**